packages/components/src/Persona.purs

```diff
@@ -605,8 +586,61 @@ getPayments uuid auth =
 type Forbidden = ServerError
   ( forbidden :: { description :: String } )
 
+type ApiJanrainUser =
+  { uuid        :: UUID
+  , email       :: Nullable String
+  , firstName   :: Nullable String
+  , lastName    :: Nullable String
+  , consent     :: Array GdprConsent
+  , legal       :: Array LegalConsent
+  , cusno       :: Nullable String
+  , otherCusnos :: Nullable (Array String)
+  }
+
+type ApiFaroUser =
+  { cusno         :: Cusno
+  , name          :: String
+  , address       :: Nullable Address
+  , email         :: Nullable String
+  , subscriptions :: Nullable (Array Subscription)
+  }
+
+type ApiSearchResult =
+  { janrain     :: Nullable ApiJanrainUser
```

Comment:
I now wish we had called these `janrainUser` and `faroUsers` 😁 Well, maybe we'll change them at some point.